ChatCore/Models/LoadingDirection.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LoadingDirection.swift
+//  STRVChatCore
+//
+//  Created by Daniel Pecher on 22.02.2021.
+//
+
+import Foundation
+
+public enum LoadingDirection {
```

Comment:
I can't build the example project because this file is not part of `ChatCore` framework